src/Microsoft.TestPlatform.CrossPlatEngine/DataCollection/InProcDataCollector.cs

```diff
@@ -98,7 +94,7 @@ private static MethodInfo GetMethodInfoFromType(Type type, string funcName, Type
             MethodInfo methodInfo = null;
 
             var typeInfo = type.GetTypeInfo();
-            methodInfo = typeInfo?.GetMethod(funcName, argumentTypes);
+            methodInfo = typeInfo.AsType()?.GetMethod(funcName, argumentTypes);
```

Comment:
In NS1.4 TypeInfo does not contain definition for GetMethod, or GetConstructor.
Although Type will continue to contain definition GetMethod, & GetConstructor, Will confirm with Immo that whether this is correct or not.